code/datums/riding.dm

```diff
@@ -275,4 +275,58 @@
 		handle_vehicle_layer()
 		handle_vehicle_offsets()
 	else
-		user << "<span class='notice'>You'll need something  to guide the [ridden.name].</span>"
\ No newline at end of file
+		user << "<span class='notice'>You'll need something  to guide the [ridden.name].</span>"
+
+//CYBORGS. NO, THEY ARE NOT ANIMALS.
+/datum/riding/cyborg
+	keytype = null
+	vehicle_move_delay = 1
+
+/datum/riding/cyborg/proc/ride_check(mob/user)
```

Comment:
It needs to check CONSTANTLY so a borg can't keep someone who gets stunned ontop of them.